plugins/kubernetes/app/models/kubernetes/role_config_file.rb

```diff
@@ -36,7 +29,10 @@ def parse_replication_controller
 
     def parse_service
       service_hash = as_hash('Service')
-      @service = Service.new(service_hash) unless service_hash.nil?
+      @service = Service.new(service_hash, :recurse_over_arrays => true) unless service_hash.nil?
```

Comment:
Adding `recurse_over_arrays` here is obsolete as the `Service` initializer handles that.